spec/Vehicle/Vehicle.vspec

```diff
@@ -261,14 +264,14 @@ RoofLoad:
   unit: kg
   description: The permitted total weight of cargo and installations (e.g. a roof rack) on top of the vehicle.
 
-cargoVolume:
+CargoVolume:
   datatype: float
   type: attribute
   unit: l
   description: The available volume for cargo or luggage. For automobiles, this is usually the trunk volume.
   min: 0
 
-emissionsCO2:
+EmissionsCO2:
   datatype: int16
   type: attribute
   description: The CO2 emissions.
```

Comment:
For later: Needs better description

Reply:
Agree, I assume it should be linked to some standard. 

But is there possibly multiple standards? (e.g. EU vs US)  And do we need to handle different versions of them? E.g. an older car might report emissions according to an older standard, possibly not directly comparable to the value from a newer standard. Do we need/want multiple signals, one for each standard, so that a vehicle possibly can report values according to multiple standards?

Reply:
I just _assume_ this might be CO2/km, which will be given by car manufactuers according to some standard, and it also is used for taxation purposes in Germany.
That would be a useful attribute, but then we should describe it accordingly (and uint16 might be more appropriate).

We just need to make sure this isn't anything else :)